dataactvalidator/scripts/loadAgencies.py

```diff
@@ -139,7 +139,7 @@ def delete_missing_sub_tier_agencies(models, new_data):
 def update_sub_tier_agencies(models, new_data, cgac_dict, frec_dict):
     """Modify existing models or create new ones"""
     for _, row in new_data.iterrows():
-        if row['cgac_code'] not in cgac_dict or row['frec_code'] not in frec_dict:
+        if row['cgac_code'] not in cgac_dict:
```

Comment:
Why did we remove the frec check? Wouldn't we want to change it to an "and" instead of removing the check entirely?